nasa/harmony
# TRADE-138 - Configure batchee stitchee for testing

## Jira Issue ID

TRADE-138

## Description

This PR adds configurations for the Batchee>Stitchee>Concise, and the L2ss>Batchee>Stitchee>Concise service chains.

## Local Test Steps

- Pull this branch of Harmony locally.
- Pull the Docker images for Batchee and Stitchee from GitHub, or build them locally from the HEAD of main for the Batchee and Stitchee repositories.
- Run through the regression tests locally, to make sure the service gets invoked.

## PR Acceptance Checklist
